### ISSUE TITLE
Custom search bug - multiple children with the same key

### ISSUE THREAD

**Issue by dsego** [komarovalexander/ka-table#102]
Modified [search example ](https://komarovalexander.github.io/ka-table/#/search) to include a participant named "Falsey False".
Searching for the score "false" produces a bug, where the row with Falsey False is shown twice.

See here:
https://stackblitz.com/edit/table-search-js-bbjoov?file=Demo.js

Console error:
```
Warning: Encountered two children with the same key, `7`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
    in Rows (created by VirtualizedRows)
    in VirtualizedRows (created by TableBodyContent)
    in TableBodyContent (created by TableBody)
    in tbody (created by TableBody)
    in TableBody
    in table
    in div
    in Unknown
    in div
    in Unknown (created by SearchDemo)
    in SearchDemo (created by App)
    in div (created by App)
    in App react-dom.development.js:70:42

```

**Comment by komarovalexander** [comment]
Hi @dsego 
Good catch!
Will fix it soon

**Comment by komarovalexander** [comment]
The issue was fixed in 6.0.1 version

Example: https://stackblitz.com/edit/table-search-js-bbytq6?file=Demo.js
